.gitignore

```diff
@@ -10,3 +10,5 @@ examples/transitions
 examples/users
 examples/switch
 examples/message
+examples/fillresizer
+.*
```

Comment:
Why is this needed?

Reply:
I mean .* will ignore all files/folders starting with a '.' , but why?

Reply:
Also `examples/fillresizer` is a linux binary and should not be in this PR.

Reply:
in my project has a .project file, and I just want ignore it.  Now I have remove .* from .gitignore file. examples/fillresizer I also remove from pr.  thanks for you comments.